nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/AddPermission.java

```diff
@@ -136,4 +140,12 @@ protected void addPermission(DocumentModel doc) {
         }
     }
 
+    /**
+     * @since 8.10
+     */
+    private boolean isUserFound(String username) {
```

Comment:
I did this because I found the method name `isUserFound` more semantic, further more, it allows to keep the version since which the check is added. But based on your feedback, adding a version `@since` should be unnecessary ...